### DISCUSSION TITLE
How many ⬛◼️ are there?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#951] [Galaxy Trivia] [answered]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/e4742976-c51a-4e6a-bc74-17e534320782)

**Comment by MrAlaminH** [comment, marked as answer]
18 squares, 6 squares in the middle, with another 12 squares around the outside.

**Comment by peterbaccaro** [comment]
There are 18 squares in total.

There are 6 squares in the middle, with another 12 squares around the outside.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/b3254e22-bd79-410e-86a0-525cee362977)

**Comment by tomerf** [comment]
There are 18 squares, 11 straight squares and 7 rotates to some degree (not 90°) (the ones with the white edges).
There are many more smaller _rectangles_ created from the overlap of the big squares, some of them might even make a perfect square themselves, probably around the middle left, but it's hard to tell.
